### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by josipkh**
This PR fixes a few small typos in the MATLAB examples and the website.

### CODE REVIEW

**Review by FreyJo — APPROVED** [review]
Thank you :)

File: examples/acados_matlab_octave/test/test_all_examples.m
Comment (FreyJo) [inline examples/acados_matlab_octave/test/test_all_examples.m:38]: It seems like this is failing.
But the fix might be contained in https://github.com/acados/acados/pull/1320/files

Comment (josipkh) [reply examples/acados_matlab_octave/test/test_all_examples.m:38]: Yes, I think I will wait until it's merged :)

**Pull request merged**
